features/apps/static.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Static
+
+  # TODO: EXPORT this test as it does not meet the eligibility
+  # criteria in docs/writing-tests.md.
+  #
+  # - Covers site-wide config: N (not applicable)
+  # - Targets data transfer: N (not applicable)
+  # - Second critical check: N (not tested in app)
+  #
+  # Should be tested in Static [^1].
+  #
+  # [^1]: https://github.com/alphagov/static/blob/9f835b87a3aa6ed867b98b2f4534dd64c9510626/app/views/root/_gem_base.html.erb#L43
+  #
+  Scenario: Check the crown logo links to GOV.UK homepage
+    When I visit "/"
+    Then the logo should link to the homepage
```

Comment:
Can we consider deleting this test? It seems quite arbitrary. I notice the feature is tested in govuk_publishing_components: https://github.com/alphagov/govuk_publishing_components/blob/efea45dcbad13766ba73b028cb6cc3a2f99ce34c/spec/components/layout_super_navigation_header_spec.rb#L89-L93

So if we were to keep it, it'd probably count as a second critical check - but I don't think it's needed.

Reply:
Ah, are you meaning that this is a test that static renders the gem_layout? 

It seems a pretty tenuous thing to test as we're basically just testing that a [variable set in static](https://github.com/alphagov/static/blob/577fd669535a8f5e44e26c2764e54a63ff51b793/app/views/root/_gem_base.html.erb#L4) gets past through a couple of layers of components (layout_for_public -> layout_super_navigation_header). 

I'd suggest it can be removed as global layout is covered implicitly in the [cookie banner](https://github.com/alphagov/smokey/blob/52f3f0cd73d56f9c7007b0c026109c7d568948db/features/apps/frontend.feature#L12-L15).